async-nats/src/jetstream/stream.rs

```diff
@@ -1220,13 +1220,13 @@ fn is_default<T: Default + Eq>(t: &T) -> bool {
 #[derive(Debug, Default, Serialize, Deserialize, Clone, PartialEq, Eq)]
 pub struct ClusterInfo {
     /// The cluster name.
-    #[serde(default)]
+    #[serde(default, skip_serializing_if = "Option::is_none")]
```

Comment:
Instead this, we can remove `Serialise` from fields that are never actually serialized (as they're only for deserializing fields received from the server).

Reply:
Yeah, you're 100% right, not sure how I missed that. I've changed the approach and made a bunch of changes.